src/Html.php

```diff
@@ -1956,9 +1957,16 @@ public static function getNonArrayableName(string $name): string
      * Render attribute in HTML tag.
      *
      * @link https://html.spec.whatwg.org/#a-quick-introduction-to-html
+     * @throws ValueError When name contains any of the following: SPACE " ' > / =
      */
     private static function renderAttribute(string $name, string $encodedValue = '', string $quote = '"'): string
     {
+        if (preg_match('/[\s"\'>\/\=]/', $name) > 0) {
+            throw new ValueError(
```

Comment:
We were using `InvalidArgumentException` for such cases previously. I think it's better to be consistent.

Reply:
Does this regexp cover all deny characters from docs: https://html.spec.whatwg.org/multipage/syntax.html#attributes-2 ?